README.md

```diff
@@ -42,6 +43,6 @@ Ideally, you need two hosts to run this project:
 
 Now you can proceed to any of the following tasks:
 
- * [Cloning a Satellite server](docs/satellite-clone.md)
+ * [Cloning a Satellite server (or) Migrating a Satellite server from rhel6 to rhel7](docs/satellite-clone.md)
```

Comment:
Migration process is a little bit different from the cloning process - Migration needs postgres dumps (candlepin.dump and foreman.dump) where as Cloning requires pgsql_data.tar.gz